src/dnfile/__init__.py

```diff
@@ -118,7 +117,7 @@ def dump_info(self, dump=None, encoding="utf-8"):
                                 ("RowSize", t.row_size),
                             ):
                                 dump.add_line(
-                                    "{0:<20}{1}".format(label+":",str(value)),
+                                    "{0:<20}{1}".format(label  + ":", str(value)),
```

Comment:
we can optionally suppress this style change, too. its new to me, but your the author, so happy to tweak the config if you like the existing style.